src/Rings/mpoly-localization_types.jl

```diff
@@ -178,6 +178,7 @@ mutable struct MPolyComplementOfKPointIdeal{
     length(a) == ngens(R) || error("the number of variables in the ring does not coincide with the number of coordinates")
     n = length(a)
     kk = coefficient_ring(R)
+    @req kk isa Field "This localization is currently only available over fields"
```

Comment:
```suggestion
    @req kk isa Field "This localization is only available over fields"
```
Since we do not plan to change this in the future, it is removed here.

Reply:
done